Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "supernova"
 version = "0.1.0"
 authors = ["0xazure <stuart.crust@gmail.com>"]
+license = "Apache-2.0 OR MIT"
```

Comment:
I'm not sure if Cargo will support multiple license files - see https://github.com/rust-lang/cargo/issues/5933

Reply:
This is the format [`cargo` uses](https://github.com/rust-lang/cargo/blob/e07088baceb70e570977cc6d01b53c492b398f0a/Cargo.toml#L7) to specify the licenses.

rust-lang/cargo#5933 refers to the `licence-file` key, which [according to the documentation](https://doc.rust-lang.org/cargo/reference/manifest.html#package-metadata) says:

> If a project is using a nonstandard license, then this key may be specified in lieu of the above key and must point to a file relative to this manifest (similar to the readme key).

which I don't think applies because these licenses _are_ standard.

Further, the `license` key [_does_ support](https://doc.rust-lang.org/cargo/reference/manifest.html#package-metadata) multiple values:

> Multiple licenses can be separated with a `/`, although that usage is deprecated. Instead, use a license expression with AND and OR operators to get more explicit semantics.

Reply:
Ah great catch - I completely misunderstood! 😅